tools/dict

```diff
@@ -114,24 +186,39 @@ unicode
 updateable
 upfront
 urireference
+uris
 uritemplate
 url
 urls
+username
 utc
 utf
 uuid
+validator
 versioned
 versionid
 versioning
 versionscount
 versionsurl
 vid
+webhook
+websockets
+wildcards
+willmessage
+willtopic
 wip
+wss
+www
 xml
+xmlschema
+xpath
 xreg
 xregbasicmodel
 xregfullmodel
 xregistries
 xregistry
+xregsitry
```

Comment:
Really?

Reply:
(These are typos on lines 218 and 293 of the new version of schema/spec.md)

Reply:
yea, this is why I need to you remove my stuff :-)
thanks!